srcpkgs/qhull/template

```diff
@@ -1,16 +1,21 @@
 # Template file for 'qhull'
 pkgname=qhull
-version=2019.1
+version=2020.2
 revision=1
 build_style=cmake
+make_build_target="all libqhull"
 short_desc="Computes volumes, surface areas and approximations to the convex hull"
 maintainer="Martin Riese <grauehaare@gmx.de>"
-license="BSD"
+license="Qhull"
 homepage="http://www.qhull.org"
+changelog="http://www.qhull.org/src/Changes.txt"
 distfiles="https://github.com/qhull/qhull/archive/$version.tar.gz"
-checksum=cf7235b76244595a86b9407b906e3259502b744528318f2178155e5899d6cf9f
+checksum=59356b229b768e6e2b09a701448bfa222c37b797a84f87f864f97462d8dbc7c5
 
 post_install() {
+	# Manually install libqhull (deprecated by the package author)
+	cp -r ${wrksrc}/build/libqhull.so* ${DESTDIR}/usr/lib
+
 	install -Dm644 ${FILESDIR}/qhull.pc ${DESTDIR}/usr/lib/pkgconfig/qhull.pc
 	sed -i "s|@VERSION@|${version}|" ${DESTDIR}/usr/lib/pkgconfig/qhull.pc
 	# convenience links for backwards compatibility
```

Comment:
Can you confirm that the "compatibility links" in in `/usr/include` are necessary to build the dependants? If not, they should be dropped.

Reply:
Yes, required for `octave`.